projects/isy-angular-widgets/assets/theme/bootstrap4/bootstrap4-light/_variables_light.scss

```diff
@@ -849,8 +850,10 @@ $imagePreviewActionIconFontSize: 1.5rem !default;
 $imagePreviewActionIconBorderRadius: 50% !default;
 
 :root {
-  font-family: BundesSans, 'Liberation Sans', sans-serif;
-  --font-family: BundesSans, 'Liberation Sans', sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji',
+    'Segoe UI Emoji', 'Segoe UI Symbol';
+  --font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif,
+    'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
   --surface-a: #{$shade000};
```

Comment:
Warum gibt es Änderungen in dieser Datei? Das sollte m.E. über das IsyFact Themen überschrieben werden

Reply:
Die variablen wurden früher in der ReadMe aufgelistet, dass diese im Bootstrap Theme von uns geändert wurden. Das ist jetzt wegen dem Auslagern in das IsyFact Theme wieder auf den Default Wert gesetzt worden.